src/Shell/RoutesShell.php

```diff
@@ -34,12 +34,13 @@ class RoutesShell extends Shell
      */
     public function main()
     {
-        $output = [];
+        $output = [
+            ['Route name', 'URI template', 'Defaults']
+        ];
         foreach (Router::routes() as $route) {
             $output[] = [$route->getName(), $route->template, json_encode($route->defaults)];
         }
-
-        $this->_outWithColumns($output);
+        $this->_io->macro('table', $output);
```

Comment:
@markstory I've tried to replace this line with
```php
$this->_io->table($output);
```
as I understood it was working but it's no as it seems to be an extra array around args. They should be the same right or am I doing something wrong ?